src/tox/config/__init__.py

```diff
@@ -270,6 +270,10 @@ def parseconfig(args, plugins=()):
         pm.hook.tox_configure(config=config)  # post process config object
         break
     else:
+        if "--help" in args or "-h" in args:
```

Comment:
shouldn't we extract this from the arg parse; I mean this solution will not work with ``tox -ah`` 🤔

Reply:
ooh i forgot to pass through arg sparser..

Reply:
solved. thank you :smiley: